deploy/rhsm-clowdapp.yaml

```
@@ -12,6 +12,25 @@ parameters:
     value: quay.io/cloudservices/rhsm-subscriptions
   - name: IMAGE_TAG
     value: latest
+  - name: SWATCH_UNLEASH_IMPORT_IMAGE
+    value: quay.io/cloudservices/swatch-unleash-import
+  - name: SWATCH_UNLEASH_IMPORT_IMAGE_TAG
+    value: latest
+  - name: SWATCH_UNLEASH_IMPORT_MEMORY_REQUEST
+    value: 10Mi
```

```suggestion
    value: 25Mi
```

I was getting `forbidden: memory max limit to request ratio per Pod is 2, but provided ratio is 5.000000` and had to drop the `--no-remove-resources app:rhsm` from the bonfire command to get it to work.  Probably just need to

---

DONE.

I bumped it and had no issues. Please confirm.